drivers/flash/flash_stm32_qspi.c

```diff
@@ -466,7 +696,11 @@ __weak HAL_StatusTypeDef HAL_DMA_Abort_IT(DMA_HandleTypeDef *hdma)
 /*
  * Transfer Error callback.
  */
-void HAL_QSPI_ErrorCallback(QSPI_HandleTypeDef *hqspi)
+#ifndef STM32_OSPI_MIMIC_QSPI
+void HAL_QSPI_ErrorCallback(QSPI_HandleTypeDef * hqspi)
+#else
+void HAL_OSPI_ErrorCallback(OSPI_HandleTypeDef *hqspi)
```

Comment:
Since these methods are defined in a header file, they shouldn't be renamed right?